packages/usa-modal/src/usa-modal.twig

```diff
@@ -17,6 +17,34 @@
         {% if description.text -%}
         <div class="usa-prose">
           <p id="{{ description.id }}">{{ description.text }}</p>
+          {% if nestedForms -%}
```

Comment:
Let's delete and create a simpler twig test template.

Reply:
Resolved in [a93f022](https://github.com/uswds/uswds/pull/5049/commits/a93f022bfcb154c7665861b98338d2e482f5b13b)